wpeverest/everest-forms
# Tweak - Review notice

### All Submissions:






### Changes proposed in this Pull Request:



Previously the content and the design is not appropriate, so the content and the design have been changed in this PR.

### How to test the changes in this Pull Request:

1. Verify if the contents and design of the review notice have been changed.
https://prnt.sc/15ovwir

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Review notice
